search/search-domain/src/main/java/io/camunda/search/entities/RoleEntity.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record RoleEntity(Long roleKey, String name, Long entityKey) {}
```

Comment:
💭 I think here `entityKey` is not necessary, in the Zeebe record we have that as a field to provide a key of an entity to assign/remove from the role. IMO when we export the record to ES we will actually add the entity to a list of assigned resources.

This indicates to me that we would more likely have a format like
`Long roleKey, String name,Set<String> assignedUserKeys ...`

Reply:
I've changed `RoleEntity` to exactly that!